### PR TITLE
Add french translations

### PR DESCRIPTION
**Pull request by amirtds**

### CODE REVIEW

File: gradefetcher/gradefetcher.py [hunk 1]
```diff
@@ -264,20 +283,23 @@ def grade_user(self, data, suffix=""):
...
+                                reason = self.i18n_service.gettext(
+                                    "Assignment {assignment_id}: <b>Passed</b>").format(
+                                        assignment_id=result["assignment_id"],
+                                    )
                                 reasons.append(reason)
-                            elif result["grade"] == 0:
-                                reason = "Assignment {assignment_id}: <b>Failed</b> - {reason} ".format(
+                            elif result["grade"] == 0: 
+                                reason_api_text = self.i18n_service.gettext(result["reason"])
+                                reason = self.i18n_service.gettext("Assignment {assignment_id}: <b>Failed</b> - {reason_api_text}").format(
```
Comment (bryanlandia) [inline gradefetcher/gradefetcher.py:293]: @amirtds I wonder if you should join the two `gettext`s here so it's easier to translate... 
like:

`self.i18n_service.gettext("Assignment {assignment_id}: <b>Failed</b> - ".format(.... )) + reason_api_text`

File: gradefetcher/gradefetcher.py [hunk 2]
```diff
@@ -304,15 +326,9 @@ def grade_user(self, data, suffix=""):
...
-        <span>
-        You got <span class="grade">{grade}% </span> score for this activity.<br />
-        Explanation: <span class="reason"><ul>{reasons_msg}</ul></span>
-        <span>
-        """.format(
-            grade=grade, reasons_msg=reasons_msg
-        )
+            reasons_msg += "<li>{reason}</li>".format(reason=reason)                
+        self.htmlFormat = self.i18n_service.gettext("You got <span class='grade'>{grade}% </span> score for this activity.<br />Explanation: <span class='reason'><ul>{reasons_msg}</ul></span>").format(
+            grade=grade, reasons_msg=reasons_msg)
```
Comment (bryanlandia) [inline gradefetcher/gradefetcher.py:331]: I wonder if it would be better to do the gettext and then apply the format to it to add the `<span...` etc.

**Review by bryanlandia — APPROVED** [review]
It looks good, just a couple of suggestions that aren't critical or at least could be done later.

**Comment by amirtds** [comment]
Thanks @bryanlandia I created another PR from this branch #7 I address your comments in new PR to prevent conflicts

**Pull request merged**